sources/packages/backend/apps/queue-consumers/src/processors/schedulers/cra-integration/cra-response-integration.scheduler.ts

```diff
@@ -23,18 +26,20 @@ export class CRAResponseIntegrationScheduler extends BaseScheduler<void> {
    * @params job job details.
    * @returns Summary with what was processed and the list of all errors, if any.
    */
-  @Process()
-  async processResponses(job: Job<void>): Promise<ProcessResponseQueue[]> {
-    this.logger.log(
-      `Processing CRA integration job ${job.id} of type ${job.name}.`,
-    );
+  protected async process(
```

Comment:
Please add `processSummary` to comments.

Reply:
Added!